smoke_test.sh

```diff
@@ -0,0 +1,27 @@
+# Spin up iqe pod and execute IQE tests in it
+
+# Env vars defined by caller:
+#IQE_PLUGINS="plugin1,plugin2" -- pytest plugins to run separated by ","
+#IQE_MARKER_EXPRESSION="mymarker" -- pytest marker expression
+#IQE_FILTER_EXPRESSION="something AND something_else" -- pytest filter, can be "" if no filter desired
+#NAMESPACE="mynamespace" -- namespace to deploy iqe pod into, can be set by 'deploy_ephemeral_env.sh'
+
+#IQE_POD_NAME="iqe-tests"
+
+# create a custom svc acct for the iqe pod to run with that has elevated permissions
+# SA=$(oc get -n $NAMESPACE sa iqe --ignore-not-found -o jsonpath='{.metadata.name}')
+# if [ -z "$SA" ]; then
+#     oc create -n $NAMESPACE sa iqe
+# fi
+# oc policy -n $NAMESPACE add-role-to-user edit system:serviceaccount:$NAMESPACE:iqe
+# oc secrets -n $NAMESPACE link iqe quay-cloudservices-pull --for=pull,mount
+oc apply -f $APP_ROOT/deploy/rbac-cji-smoketest.yml
+sleep 30
```

Comment:
:raised_eyebrow: tell me more about this :P 

Are we waiting on the job to appear here or something, is there a more efficient way to wait here?

Reply:
Yeah I'm working on figuring that out now. We need the job to exist, but we have to wait for clowder to form and create it first. The solution in the kuttl tests is `sleep 60` so I'm trying to find a dynamic way to get notified that the job is ready and running.

Reply:
Would be far more efficient to do in python I guess... but I think something along these lines could work in bash (need to make sure the script is using `#!/bin/bash` to get `$SECONDS` -- see https://stackoverflow.com/a/11198713)

```bash
#!/bin/bash
job_name=foo
found=0
end=$((SECONDS+30))

echo "Waiting for Job $job_name to appear"

while [ $SECONDS -lt $end ]; do
    if `oc get job $job_name >/dev/null 2>&1`; then
        found=1
        break
    fi
done

if ! found; then
    echo "Job $job_name failed to appear"
    exit 1
fi
```

Reply:
Oops... forgot you'll want a `sleep 1` at the bottom of that while loop too ;)